package.json

```diff
@@ -21,7 +21,8 @@
     "prepublishOnly": "turbo run prepublishOnly --parallel",
     "typecheck": "turbo run typecheck --parallel",
     "clean": "turbo run clean --parallel && yarn clean:root",
-    "clean:root": "rm -rf node_modules"
+    "clean:root": "rm -rf node_modules",
+    "knip": "npx -y knip --no-gitignore"
```

Comment:
I think rather than `npx` you should do `yarn knip --no-gitignore`

Reply:
Will do once this gets resolved: https://github.com/webpro/knip/issues/251